app/models/project.rb

```diff
@@ -13,13 +13,17 @@ class Project < ActiveRecord::Base
 
   after_create :create_template_defaults
 
-  attr_accessible :name, :description, :template, :archived
+  attr_accessible :name, :description, :template, :archived, :extensions
 
   scope :active,
     :include    => [:users, {:stages => [:hosts, :recipes]}, :configuration_parameters, :user_project_links],
     :conditions => {:archived => false},
     :order      => "projects.name ASC"
 
+  serialize :extensions
```

Comment:
I figured that since it is only a small number of gems that capistrano uses and that each project would most likely use no more than four at a time and that the list of gems would remain pretty constant it would be easier to use serialization.  That and the main function of including the extensions to this point was iterating through the array and appending the extensions' strings to `require 'capistrano/'` which was easily done as an attribute of project.

That being said, I am open to changing extensions to a model.  What `extensions` does now can be done as a model but the level of association and functionality added might be a little overkill at this point.  That and the majority, or more likely all, of the model instances would be the default extensions below.